unit/java_bytecode/java_bytecode_parse_generics/parse_signature_descriptor_mismatch.cpp

```diff
@@ -39,18 +39,28 @@ SCENARIO(
       require_symbol::require_complete_class(inner_symbol);
   }
 
-  THEN("There is a symbol for the constructor of the inner class with correct"
-         " descriptor")
+  THEN("There is a symbol for the constructor of the class Inner with "
+         "the correct number of parameters")
   {
     const std::string func_name=".<init>";
-    const std::string func_descriptor=":(LOuter;LAbstractGenericClass;)V";
+    const std::string func_descriptor=
+      ":(LSignatureDescriptorMismatch;LAbstractGeneric;)V";
     const std::string process_func_name=
       inner_prefix+func_name+func_descriptor;
     REQUIRE(new_symbol_table.has_symbol(process_func_name));
 
     const code_typet func_code=
       to_code_type(new_symbol_table.lookup_ref(process_func_name).type);
     REQUIRE(func_code.parameters().size()==3);
+
+    // TODO: for now, the parameters are not generic because we fall back to
+    // descriptor due to mismatch; enable tests when fixed
```

Comment:
Jira reference?

Reply:
Don't think the ticket exists, if you could create it @svorenova that would be great.